src/apps/CustomSocketsRegistry/CustomSocketsRegistrySummary.js

```diff
@@ -44,69 +42,70 @@ const styles = {
   }
 };
 
-const renderCallMethodsLabel = (call) => {
-  const currentInstance = SessionStore.getInstance();
-  const methods = call.methods[0] === '*' ? ['get', 'post', 'put', 'patch'] : call.methods;
+const CustomSocketsRegistrySummary = ({ item }) => {
+  const renderCallMethodsLabel = (call) => {
+    const methods = call.methods[0] === '*' ? ['get', 'post', 'put', 'patch'] : call.methods;
 
-  return _.map(methods, (method, index) => (
-    <div
-      key={index}
-      style={styles.call}
-    >
-      <MethodLabel method={method} />
-      <div style={styles.callUrl}>
-        {`${SYNCANO_BASE_URL}/v1.1/instances/${currentInstance.name}/endpoints/sockets/${call.name}/`}
+    return _.map(methods, (method, index) => (
+      <div
+        key={index}
+        style={styles.call}
+      >
+        <MethodLabel method={method} />
+        <div style={styles.callUrl}>
+          {`${SYNCANO_BASE_URL}/v1.1/instances/${item.instanceName}/endpoints/sockets/${call.name}/`}
+        </div>
       </div>
-    </div>
-  ));
-};
+    ));
+  };
 
-const renderCalls = (calls) => (
-  _.map(calls, (call, index) => (
-    <div
-      key={index}
-      style={{ maxWidth: '100%' }}
-    >
-      {renderCallMethodsLabel(call)}
-    </div>
-  ))
-);
+  const renderCalls = (calls) => (
+    _.map(calls, (call, index) => (
+      <div
+        key={index}
+        style={{ maxWidth: '100%' }}
+      >
+        {renderCallMethodsLabel(call)}
+      </div>
+    ))
+  );
 
-const CustomSocketsRegistrySummary = ({ item }) => (
-  <div>
-    <Dialog.ContentSection>
-      <div className="col-flex-1">
-        <div style={{ fontSize: 16, lineHeight: 1.6, color: 'rgba(68,68,68, .8)' }}>
-          {`Custom Socket you just created contains several endpoints that you can connect your frontend
-            or mobile app to. Check out the their dependencies to see how everything work under the hood.
+  return (
+    <div>
+      <Dialog.ContentSection>
+        <div className="col-flex-1">
+          <div style={{ fontSize: 16, lineHeight: 1.6, color: 'rgba(68,68,68, .8)' }}>
+            {`Custom Socket you just created contains several endpoints that you can connect your frontend
+              or mobile app to. Check out the their dependencies to see how everything work under the hood.
```

Comment:
Check out ( without the ) their dependencies to see how everything WORKS (s at the end) blablabla